Meta/build-image-qemu.sh

```diff
@@ -40,6 +40,10 @@ else
     if [ ! -f "$E2FSCK" ]; then
         E2FSCK=/sbin/e2fsck
     fi
+
+    if [ ! -f "$E2FSCK" ]; then
```

Comment:
Hmm the existing pattern looks pretty fragile.

Might make sense to invert this if else chain, and use `command` instead of file checking?

Something like...

```
if command -v e2fsck ; then
    E2FSCK=e2fsck
elif command -v /sbin/e2fsck; then
    ...
```

Reply:
Yeah, good idea. However, shouldn't we look for it in `/sbin` and `/usr/sbin` before looking in the PATH? I guess `e2fsck` is more likely to be there than in the PATH in most distros. Not that such a lookup is very expensive, just wondering really.